src/main/cpp/VideoState.cpp

```diff
@@ -1120,19 +1204,13 @@ int VideoState::subtitle_thread() {
 VideoState *VideoState::stream_open(const char *filename, AVInputFormat *iformat) {
 	VideoState *is = new VideoState();
```

Comment:
For this to work we need to add (nothrow) to the new operator like this

VideoState *is = new (nothrow) VideoState();